src/cmd.rs

```diff
@@ -493,6 +502,12 @@ impl Pipeline {
         )
     }
 
+    /// Clear the internal data strcture of `Pipeline`, allowing it to be reused.
```

Comment:
Same here as before: make it more clear that it's empty afterwards.

Reply:
@badboy 
Would you be kind enough to suggest a phrasing for this?

Reply:
oh... i see you did. didn't scroll up enough...

Reply:
What about:

> Clear a `Pipeline` object internal data structure. This allows reusing a `Pipeline` object as a clear object but with a minimal amount of memory released/allocated.